packages/codegen/src/generateEntityCodegenFile.ts

```diff
@@ -96,6 +97,10 @@ export function generateEntityCodegenFile(config: Config, dbMeta: DbMetadata, me
     }
 
     let setter: Code | string;
+    if (p.columnName === "bio") {
```

Comment:
Nit: snuck in

Reply:
Drats! Good spot. Wasn't sure if columnDefault was going to be "" or "''".